deploy/helm-chart/templates/deployment-promscale.yaml

```diff
@@ -39,16 +39,10 @@ spec:
         - image: {{ .Values.image }}
           imagePullPolicy: {{ .Values.imagePullPolicy }}
           name: promscale-connector
-          {{- if or (.Values.openTelemetry.enabled) (.Values.extraArgs) }}
```

Comment:
Do we not still need the check for `.Values.extraArgs`? I assume that if there are none set, the resulting args block is empty.